docker-compose-dev.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
+
+services:
+    mongodb:
+        image: mongo
+        command: [ "/usr/bin/mongod", "--replSet", "projectsRepSet" ]
+        ports:
+            - "27017:27017"
+        restart: "on-failure"
+
+    mongodb_replica_setup:
+        build: ./backend/mongodb_replica_setup
+        depends_on:
+            - "mongodb"
+
+    elasticsearch:
+        image: docker.elastic.co/elasticsearch/elasticsearch:5.4.1
+        environment:
+            - "http.host=0.0.0.0"
+            - "transport.host=127.0.0.1"
+            - "xpack.security.enabled=false"
+            - "action.auto_create_index=true"
+        ports:
+            - "9200:9200"
+        restart: "on-failure"
+
+    mongo-connector:
+        build: ./backend/mongo_connector
+        restart: "always"
+        depends_on:
+            - "elasticsearch"
+
+    flask:
+        build:
+            context: ./
+            dockerfile: ./Dockerfile-dev-backend
+        restart: "always"
+        volumes:
+            - ./backend:/backend
+
+    yarn:
+        build:
+            context: ./
+            dockerfile: ./Dockerfile-dev-frontend
+        restart: "always"
+        depends_on:
+            - "flask"
+        volumes:
+            - ./web:/frontend
+        ports:
+            - "3000:3000"
```

Comment:
Well it's hard to correctly mark the part I meant so it kind of landed in the nowhere. What I meant is also specifying the FE part in the compose yaml and mounting the volumes for instant code changes. We already had that in the BE once and it was very convenient. Also having a shared port to access the FE specifically, just better overall.